client/a8c-for-agencies/components/pressable-usage-details/index.tsx

```diff
@@ -101,16 +97,8 @@ export default function PressableUsageDetails( { existingPlan }: Props ) {
 						<div className="pressable-usage-details__info-top-right">
 							{ translate( '%(visits_count)s of %(max_visits)s', {
```

Comment:
Anyone can help confirm this? I couldn't get to it to test 🙏

Reply:
Shows `0 de 50.000` instead of `0 de 50,000` now 👍 
<img width="1120" alt="Screenshot 2025-02-03 at 11 02 24 AM" src="https://github.com/user-attachments/assets/226dc406-62b2-411c-84c6-1707ef41511e" />